### PR TITLE
Add clarifications, fix typos for Dev Container Features proposal

### PR DESCRIPTION
**Pull request by Chuxel**
There was some missing info and typos I corrected.  I also linked out to the new feature-template and features repositories. //cc @bamurtaugh

### CODE REVIEW

**Comment by Chuxel** [comment]
Actually @joshspicer @jkeech @edgonmsft - This still refers to the concept of a collection. Is that still valid?  If so its missing from the template.  https://github.com/devcontainers/feature-template

File: proposals/devcontainer-features-distribution.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-This specification defines a pattern where community members and organizations can author and self-publish [dev container 'features'](./devcontainer-features.md). 
+This specification defines a pattern where community members and organizations can author and self-publish [Dev Container Features](./devcontainer-features.md). 
 
 Goals include:
 
-- For community authors, a "self-served" mechanism for dev container feature publishing, either publicly or privately.
-- For users, the ability to validate the integrity of previously fetched assets. 
-- For users, the ability for a user to pin to a particular version (absolute, or semantic version) of a feature to allow for consistent, repeatable environments.
-- The ability to standardize publishing such that [supporting tools](../docs/specs/supporting-tools.md) may implement mechanisms for feature discoverability.
+- For Feature authors, create a "self-service" way to publish a Feature, either publicly or privately, that does not centrally controlled.
```
Comment (bamurtaugh) [inline proposals/devcontainer-features-distribution.md:7]:
```suggestion
- For Feature authors, create a "self-service" way to publish a Feature, either publicly or privately, that is not centrally controlled.
```

File: proposals/devcontainer-features-distribution.md [hunk 2]
```diff
@@ -123,13 +128,13 @@ oras push ${REGISTRY}/${NAMESPACE}:latest \
...
-### Directly Reference Tarball
+### Directly referencing a tarball
 
-A feature can be referenced directly in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) file by HTTPS URI that points to the tarball from the [package step](#packaging).
+A Feature can be referenced directly in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) file by HTTPS URI that points to the tarball from the [package step](#packaging).
 
 The `.tgz` archive file must be named `devcontainer-feature-<featureId>.tgz`.
 
-### Addendum: Locally Referenced
+### Locally referenced features
```
Comment (bamurtaugh) [inline proposals/devcontainer-features-distribution.md:137]:
```suggestion
### Locally referenced Features
```

File: proposals/devcontainer-features.md [hunk 3]
```diff
@@ -120,40 +137,40 @@ The `id` format specified dictates how a supporting tool will locate and downloa
...
+By default, Features are installed on top of a base image in an order determined as optimal by the implementing tool.
 
 If any of the following properties are provided in the feature's `devcontainer-feature.json`, or the user's `devcontainer.json`, the order indicated by these propert(ies) are respected (with decreasing precedence).
 
-1. The `overrideFeatureInstallOrder` property in user's `devcontainer.json`. Allows users to control the order of execution of their features.
+1. The `overrideFeatureInstallOrder` property in user's `devcontainer.json`. Allows users to control the order of execution of their Features.
 2. The `installsAfter` property defined as part of a feature's `devcontainer-feature.json`.
 
-#### (1) overrideFeatureInstallOrder
+#### (1) The `overrideFeatureInstallOrder` devcontainer.json property
```
Comment (bamurtaugh) [inline proposals/devcontainer-features.md:167]: Do we need to say it's a devcontainer.json property? Could we just say:
```suggestion
#### (1) The `overrideFeatureInstallOrder` property
```

File: proposals/devcontainer-features.md [hunk 4]
```diff
@@ -120,40 +137,40 @@ The `id` format specified dictates how a supporting tool will locate and downloa
...
+Any un-versioned feature IDs listed in this array will be installed before all other Features, in the provided order. Any omitted Features will be installed in an order selected by the implementing tool, or ordered via the `installsAfter` property _after_  any Features listed in the `overrideFeatureInstallOrder` array, if applicable. 
 
-All unversioned feature `id`s provided in `overrideFeatureInstallOrder` must also exist in the `features` property of a user's `devcontainer.json`. For instance, all the features which follows the OCI registry format would include everything except for the label that contains the version (`<oci-registry>/<namespace>/<feature>` without the `:<semantic-version>`.
+All un-versioned feature `id`s provided in `overrideFeatureInstallOrder` must also exist in the `features` property of a user's `devcontainer.json`. For instance, all the Features which follows the OCI registry format would include everything except for the label that contains the version (`<oci-registry>/<namespace>/<feature>` without the `:<semantic-version>`.
```
Comment (bamurtaugh) [inline proposals/devcontainer-features.md:173]:
```suggestion
All un-versioned Feature `id`s provided in `overrideFeatureInstallOrder` must also exist in the `features` property of a user's `devcontainer.json`. For instance, all the Features which follows the OCI registry format would include everything except for the label that contains the version (`<oci-registry>/<namespace>/<feature>` without the `:<semantic-version>`.
```

File: proposals/devcontainer-features.md [hunk 5]
```diff
@@ -168,17 +185,17 @@ Example:
 
 | Property | Type | Description |
 | :--- | :--- | :--- |
-| overrideFeatureInstallOrder | array | Array consisting of the feature `id` (without the semantic version) of features in the order the user wants them to be installed.   |
+| `overrideFeatureInstallOrder` | array | Array consisting of the feature `id` (without the semantic version) of Features in the order the user wants them to be installed.   |
 
-#### (2) installsAfter
+#### (2) The `installsAfter` feature property
```
Comment (bamurtaugh) [inline proposals/devcontainer-features.md:190]:
```suggestion
#### (2) The `installsAfter` Feature property
```

File: proposals/devcontainer-features.md [hunk 6]
```diff
@@ -168,17 +185,17 @@ Example:
...
+This property is defined in an individual feature's `devcontainer-feature.json` file by the feature author.  `installsAfter` allows an author to provide the tooling hints on loose dependencies between Features.
 
-After `overrideFeatureInstallOrder` is resolved, any remaining features that declare an `installsAfter` must be installed after the features declared in the property, provided that the features have also been declared in the `features` property.
+After `overrideFeatureInstallOrder` is resolved, any remaining Features that declare an `installsAfter` must be installed after the Features declared in the property, provided that the Features have also been declared in the `features` property.
 
 | Property | Type | Description |
 | :--- | :--- | :--- |
-| installsAfter | array | Array consisting of the feature `id` that should be installed before the given feature   |
+| `installsAfter` | array | Array consisting of the feature `id` that should be installed before the given feature   |
```
Comment (bamurtaugh) [inline proposals/devcontainer-features.md:198]:
```suggestion
| `installsAfter` | array | Array consisting of the feature `id` that should be installed before the given feature   |
```
```suggestion
| `installsAfter` | array | Array consisting of the feature `id` that should be installed before the given Feature |
```

**Review by bamurtaugh — COMMENTED** [review]
Thanks for making these changes! Once they're approved by other folks too, I can open a PR to update containers.dev.

**Review by joshspicer — APPROVED** [review]

**Pull request merged**